src/routes/users.rs

```diff
@@ -22,13 +24,49 @@ pub async fn user_auth_get(
     ))
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct UserIds {
+    pub ids: String,
+}
+
+#[get("users")]
+pub async fn users_get(
+    web::Query(ids): web::Query<UserIds>,
+    pool: web::Data<PgPool>,
+) -> Result<HttpResponse, ApiError> {
+    let mut users = vec![];
+
+    for id in serde_json::from_str::<Vec<UserId>>(&*ids.ids)? {
+        let mod_data = get_user_from_id(id, &*pool).await?;
+
+        if let Some(data) = mod_data {
+            users.push(data)
+        }
+    }
```

Comment:
Same as for the `mods_get` comment;  We could either have a local function that implements it more efficiently or add implement it in the `database` module.